modules/rosa-cli-hcp-classic-examples.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// * rosa_cli/rosa-cli-permission-examples.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="rosa-cli-hcp-classic-examples_{context}"]
+= Least privilege permissions for common ROSA CLI commands
+
+The following required minimum permissions for the listed ROSA CLI commands are applicable for hosted control plane (HCP) and Classic clusters.
+
+[id="rosa-create-OIDC-providers-hcp-classic_{context}"]
+== Create an OpenID Connect (OIDC) provider
+Run the following command with the specified permissions to create your OIDC provider by using `auto` mode.
```

Comment:
Run the following command with the specified permissions to create Managed OIDC provider by using `auto` mode.